source/extensions/filters/network/thrift_proxy/decoder.h

```diff
@@ -88,7 +88,14 @@ class DecoderStateMachine : public Logger::Loggable<Logger::Id::thrift> {
   ProtocolState currentState() const { return state_; }
 
   /**
-   * Set the current state. Used for testing only.
+   * Consumes whole passthrough data without the body start.
```

Comment:
nit: I think body here is confusing. Maybe "message start" instead of "body start"?

Reply:
nice suggestion. i addressed this. thanks!